docs/.eleventy.js

```diff
@@ -179,14 +179,46 @@ module.exports = function(eleventyConfig) {
         `.trim();
     }
 
+    /**
+     * Encodes text in the base 64 format used in playground URL params.
+     * @param {string} text Text to be encoded to base 64.
+     * @see https://github.com/eslint/eslint.org/blob/1b2f2aabeac2955a076d61788da8a0008bca6fb6/src/playground/utils/unicode.js
+     * @returns {string} The base 64 encoded equivalent of the text.
+     */
+    function encodeToBase64(text) {
+        /* global btoa -- It does exist, and is what the playground uses. */
+        return btoa(unescape(encodeURIComponent(text)));
+    }
+
+    const withPlaygroundRender = {
+        render(tokens, index) {
+            if (tokens[index].nesting !== 1) {
+                return "</div>";
+            }
+
+            const state = encodeToBase64(
+                JSON.stringify({
+                    text: tokens[index + 1].content
```

Comment:
@mdjermanovic /co. from https://github.com/eslint/eslint/issues/17018#issuecomment-1523747750, is there a way you'd suggest encoding `parserOptions` settings such as `sourceType: "script"`? I've never had to do that with an ESLint configuration comment.

Reply:
These correct and incorrect fences can take in options also right? we can somehow send the sourcetype as token may be like `:::correct script`? we could actually default module or script. (which ever has higher prio and doesn't need to be mentioned always and is default) and we can override with options when needed? 🤔  if thats what you are asking for?

Reply:
That makes a lot of sense, thanks you two!